app/models/financialDetails/Charge.scala

```diff
@@ -16,57 +16,67 @@
                   transactionId: String,
                   transactionDate: Option[String] = None,
                   `type`: Option[String] = None,
                   totalAmount: Option[BigDecimal] = None,
                   originalAmount: Option[BigDecimal] = None,
-                  outstandingAmount: Option[BigDecimal] =None,
-                  clearedAmount: Option[BigDecimal] =None,
-									chargeType: Option[String] = None,
+                  outstandingAmount: Option[BigDecimal] = None,
+                  clearedAmount: Option[BigDecimal] = None,
+                  chargeType: Option[String] = None,
                   mainType: Option[String] = None,
                   items: Option[Seq[SubItem]] = None
                  ) {
   val isPaid: Boolean = {
-		outstandingAmount.fold(clearedAmount.exists(_ >= 0) && clearedAmount == originalAmount)(_ <= 0)
-	}
+    outstandingAmount.fold(clearedAmount.exists(_ >= 0) && clearedAmount == originalAmount)(_ <= 0)
+  }
 
-	def remainingToPay: BigDecimal = {
-		if (isPaid) BigDecimal(0)
-		else outstandingAmount.getOrElse(originalAmount.get)
-	}
+  def remainingToPay: BigDecimal = {
+    if (isPaid) BigDecimal(0)
+    else outstandingAmount.getOrElse(originalAmount.get)
+  }
 
   def charges(): Seq[SubItem] = items.getOrElse(Seq()).filter(_.dueDate.isDefined)
 
   def payments(): Seq[SubItem] = items.getOrElse(Seq()).filter(_.paymentReference.isDefined)
 
-  def eligibleToPay(paymentEnabled: Boolean) :Boolean = {
+  def eligibleToPay(paymentEnabled: Boolean): Boolean = {
     !isPaid && paymentEnabled
   }
 
   val due: Option[LocalDate] = charges().headOption.flatMap(_.dueDate).map(due => LocalDate.parse(due))
 
   def isOverdue: Boolean = due.exists(dueDate => dueDate.isBefore(LocalDate.now()))
 
-	def getChargeTypeKey: String = mainType match {
-		case Some("SA Payment on Account 1") => "paymentOnAccount1.text"
-		case Some("SA Payment on Account 2") => "paymentOnAccount2.text"
-		case Some("SA Balancing Charge") => "balancingCharge.text"
-		case error => {
-			Logger.error(s"[Charge][getChargeTypeKey] Missing or non-matching charge type: $error found")
-			"unknownCharge"
-		}
-	}
+  def getChargeTypeKey: String = mainType match {
+    case Some("SA Payment on Account 1") => "paymentOnAccount1.text"
+    case Some("SA Payment on Account 2") => "paymentOnAccount2.text"
+    case Some("SA Balancing Charge") => "balancingCharge.text"
+    case error => {
+      Logger.error(s"[Charge][getChargeTypeKey] Missing or non-matching charge type: $error found")
+      "unknownCharge"
+    }
+  }
+
+  val getChargeType: String = mainType match {
+    case Some("SA Payment on Account 1") => "Payment on account 1 of 2"
+    case Some("SA Payment on Account 2") => "Payment on account 2 of 2"
+    case Some("SA Balancing Charge") => "Remaining balance"
+    case error => {
+      Logger.error(s"[Charge][getChargeTypeKey] Missing or non-matching charge type: $error found")
+      "unknownCharge"
+    }
+  }
```

Comment:
As this is a funciton which is solely ever used in the audit you have created, would you be able to move this functionality into that audit model as a function instead of here?